manager/Pipfile

```
@@ -21,16 +21,17 @@ celery = "~=5.0.5"
 channels = "~=3.0.3"
 channels-redis = "~=3.2.0"
 daphne = "~=3.0.1"
-django = "~=2.2.20"
+django = "~=4.2.14"
```

```suggestion
django = "~=4.2"
```
When using `~=`, just leave the minor version blank if the package uses semantic versioning. This way, we can get security patches more easily.

https://pipenv.pypa.io/zh-cn/stable/basics.html#specifying-versions-of-a-package

I would say the same for the following.

---

Yea good catch 👍